backstop-non-responsive-ce.json

```diff
@@ -151,7 +151,7 @@
       "referenceUrl": "",
       "readyEvent": "",
       "readySelector": "",
-      "delay": 0,
+      "delay": 20000,
```

Comment:
Would it be possible to use readySelector or readyEvent instead of a delay?

Reply:
I implemented readyEvent instead of dealay, thanks